chia/wallet/nft_wallet/nft_info.py

```diff
@@ -75,6 +75,9 @@ class NFTInfo(Streamable):
     launcher_puzhash: bytes32 = LAUNCHER_PUZZLE.get_tree_hash()
     """Puzzle hash of the singleton launcher in hex"""
 
+    off_chain_data: Optional[str] = None
```

Comment:
Any objection to naming this `off_chain_metadata`? Just in case we want to support exposing other types of off-chain data at some point.